src/main.rs

```diff
@@ -1,3 +1,52 @@
+/*
+ * This file is part of Disasm-Util.
+ *
+ * Copyright 2022 SECO Mind Srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use std::env;
+use std::fs::{write, File};
+use std::io::BufReader;
+use std::path::Path;
+use std::process::exit;
+
+mod disasm;
+
+use disasm::Disasm;
+
 fn main() {
-    println!("Hello, world!");
+    let args: Vec<String> = env::args().collect();
+
+    match args.get(1) {
```

Comment:
For an command line argument parser, you can use [Clap](https://crates.io/crates/clap) crate